src/blockdata/transaction.rs

```diff
@@ -45,8 +48,26 @@ pub struct OutPoint {
     /// The index of the referenced output in its transaction's vout
     pub vout: u32,
 }
+
 serde_struct_human_string_impl!(OutPoint, "an OutPoint", txid, vout);
 
+#[cfg(feature = "schemars")]
+impl JsonSchema for OutPoint {
```

Comment:
Those can't be moved into a module, I suppose. Hmm, a bit unfortunate IMO, but I suppose it's fine.

Reply:
It depends; in this case we need to do a custom impl like this because the representation is weird. If we have per-field representations then we don't need to, but because it becomes one string we need this approach.

Reply:
Actually in some crates I have seen impl on types within the crate moved into separate modules. I thought that was not possible? Can this block be moved to a file like f.e. json_schemas.rs?

Reply:
It's cross crate impls that are banned, cross module is fine actually.

Reply:
more specifically, either the trait or the struct has to be defined in the crate for impl to be allowed.